test/fixtures/create-sarco-fixture.ts

```diff
@@ -96,7 +96,7 @@ export const createSarcoFixture = (
         archaeologistFacet as ArchaeologistFacet,
         (sarcoToken as IERC20).connect(deployer),
         diamond.address,
-        maxRewrapInterval,
+        BigNumber.from(maxRewrapInterval),
```

Comment:
Should be good with regular Number on this

Reply:
Yep, it is indeed good with regular Number, thanks. I was panic BigNumber-ing all over the place during the signature fiasco, sigh.